nats/src/main/kotlin/rys/ajaxpetproject/nats/config/NatsControllerConfigurerPostProcessor.kt

```diff
@@ -18,11 +18,14 @@ class NatsControllerConfigurerPostProcessor : BeanPostProcessor {
     }
 
     private fun <RequestT : GeneratedMessageV3, ResponseT : GeneratedMessageV3>
-            initializeNatsController(controller: NatsController<RequestT, ResponseT>, connection: Connection) {
+            initializeNatsController(
+        controller: NatsController<RequestT, ResponseT>,
+        connection: Connection
+    ) {
         connection.createDispatcher { message ->
-            val parsedData = controller.parser.parseFrom(message.data)
-            val response = controller.handle(parsedData)
-            connection.publish(message.replyTo, response.toByteArray())
+
+            ReactiveNatsHandler(controller).onMessage(message)
```

Comment:
new instance of `ReactiveNatsHandler` per every message?

Reply:
Changed.